src/selection-tools.js

```diff
@@ -249,24 +249,54 @@ html, body {
     width: 100%;
 }
 body {
+    flex: 1;
     display: flex;
     flex-direction: column;
 }
-select {
+header {
+    flex: 1;
+    padding: 8px;
+    display: none;
+}
+header button {
     width: 100%;
+    margin-top: 6px;
+}
+header button:first-child {
+    margin-top: 0;
 }
 main {
     flex: 1;
+    display: flex;
     overflow-y: auto;
     margin: 8px 0;
 }
+footer {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    width: 100%;
+}
+footer div {
+    white-space: nowrap;
+}
+footer a,
+footer a:link,
+footer a:visited,
+footer a:hover,
+footer a:active {
+    color: black;
```

Comment:
Don't hardcode the color, which would break in dark mode. Use something like `initial`, `inherit`, or `currentcolor`.

Also you can use `a:any-link`.

Reply:
Fix